### ISSUE TITLE
Cannot find images in relative path on docx book

### ISSUE THREAD

**Issue by eitsupi** [quarto-dev/quarto-cli#1019]
When I try to embed an image with a relative path from a markdown file that exists in a subdirectory as shown below, docx output on Windows seems to fail. (`quarto render --to docx`)
It was not a problem on Linux, so it may be related to the path separator character.

```sh
$ tree .
.
├── img.png
├── _quarto.yml
└── sub
    └── test.md
```

test.md's content

```md
---
title: "quarto-test"
---

## Quarto

![](../img.png)
```

**Comment by jjallaire** [comment]
I'm unable to reproduce this problem (on windows w/ identical input as shown above). I tried in three configurations:

(1) Not in a project
(2) In a project of type "default" (not a website)
(3) In a website project

In each case the docx was correctly generated.

If you are still seeing this problem please provide a git repo that has files which exhibit the problem along with the command used to render.

**Comment by eitsupi** [comment]
Thank you for confirming this. It seems to be working well on my end now too...
I may have done something wrong with the procedure. If it recurs I will check.

**Comment by eitsupi** [comment]
I am very sorry, but I misjudged the cause of the problem. The cause was the book format. (This was discovered when I inserted an image into a private HTML+docx format book).
The problem also occurred on Linux.
I am attaching a sample project.

[quarto-test.zip](https://github.com/quarto-dev/quarto-cli/files/8823397/quarto-test.zip)

**Comment by jjallaire** [comment]
Thanks again for reporting this, should be fixed here: https://github.com/quarto-dev/quarto-cli/commit/6e5f5652f844b7d80eb400af67e78ab24f4cbbe1 (fix available in build v0.9.576: https://github.com/quarto-dev/quarto-cli/releases/latest)

**Comment by eitsupi** [comment]
Just a notification: is there a possibility to remove the quarto cli workaround since this issue seems to be resolved in pandoc 2.19?
de145553da303d126fcf58294e2e2c968189be11

**Comment by jjallaire** [comment]
Thanks for the reminder! Done here: https://github.com/quarto-dev/quarto-cli/commit/5020af2dc44e170784821c0845ba6d73fbe2c4b3
